### DISCUSSION TITLE
Added Semantic HTML Merged style.css with cast.css no issues

### DISCUSSION THREAD

**Discussion by matty10s** [matty10s/group3-project#18] [Announcements]
Hey Taylor and Shannon, 

I was able to add semantic HTML to the cast.html page and merged the cast.css style sheet parts and combined into style.css without breaking anything :)

Made 2 columns.   

Hope this looks better now.

**Comment by shannonkuhn92** [comment]
Looks great! I'm gonna go in and center the images on the Trivia page. 

TAYLOR: In the mobile responsive code, I recommend changing the text from justify-aligned to normal right-aligned to prevent super weird spacing on smaller screen sizes. Please and thank you!

**Reply by shannonkuhn92** [reply]
Pictures are centered. I also added meta descriptions. Pull request is in.

**Comment by tay-baum** [comment]
Got it!! I'll go in right after work today and finish up the mobile responsive code and let ya'll know when its done! Should be around 4 EST. Hoping to get to it sooner, but not sure about work today.

P.S. looks great! The borders on the cast images look nice and clean!
